SpecFlowProjectMars1/Features/LanguageSkills.feature

```diff
@@ -0,0 +1,67 @@
+﻿Feature: Manage Languages
+  As a user
+  I want to add new languages to my profile
+  So that I can showcase the languages I know
+ @language
```

Comment:
please separate feature files into two files skills and language

Reply:
ok will separate and update